reactivesocket-cpp/src/Frame.cpp

```diff
@@ -99,6 +96,12 @@ void FrameHeader::serializeInto(folly::io::Appender& app) {
   app.writeBE<uint32_t>(streamId_);
 }
 
+void FrameHeader::serializeInto(folly::io::QueueAppender& app) {
```

Comment:
Got it to work with Writable through templates. I also had a overload of serializeInto that need not be there - removed it.